app/views/operatingsystems/_templates.html.erb

```diff
@@ -1,2 +1,2 @@
-<%= selectable_f f,:provisioning_template_id, @operatingsystem.provisioning_templates.where(:template_kind_id => f.object.template_kind_id).collect { |c| [c.name, c.id]}, { :include_blank => true }, { :label => _(f.object.template_kind.to_s) } %>
+<%= selectable_f f,:provisioning_template_id, @operatingsystem.provisioning_templates.where(:template_kind_id => f.object.template_kind_id).collect { |c| [c.name, c.id]}, { :include_blank => true }, { :label => _(f.object.template_kind.to_s), :help_inline => _(TemplatesHelper.template_help_text[f.object.template_kind.try(:name)])}  %>
```

Comment:
note we usually use `:label_help` that puts small info icon and help is displayed after click, would it work here so we are consistent across the whole app?

Reply:
Thanks for the guidelines here. New version pushed includes :label_help